front/src/modules/ui/layout/show-page/components/ShowPageSummaryCard.tsx

```diff
@@ -98,6 +98,7 @@ export const ShowPageSummaryCard = ({
       <StyledAvatarWrapper onClick={onAvatarClick}>
         <Avatar
           avatarUrl={logoOrAvatar}
+          onClick={() => console.log('on click')}
```

Comment:
can we have this method trigger the existing onAvatarClick method and remove the onClick on StyledAvatarWrapper (also onAvatarClick should be renamed handleAvatarClick)